mitosheet/src/mito/utils/actions.tsx

```diff
@@ -728,6 +774,61 @@ export const getActions = (
             searchTerms: ['export', 'download', 'excel', 'csv'],
             tooltip: "Export dataframes as a .csv or .xlsx file. Choose whether or not to include export in the code."
         },
+        [ActionEnum.Graph_SelectData]: {
+            type: 'build-time',
+            staticType: ActionEnum.Graph_SelectData,
+            iconToolbar: SelectDataIcon,
+            titleToolbar: 'Select Data',
+            longTitle: 'Select Data',
+            actionFunction: () => {
+                // We turn off editing mode, if it is on
+                setEditorState(undefined);
+
+                const currOpenTaskpane = uiState.currOpenTaskpane;
+                if (currOpenTaskpane.type === TaskpaneType.GRAPH) {
+                    const openGraph = currOpenTaskpane.openGraph;
+                    // We open the graph taskpane
+                    setUIState(prevUIState => {
+                        return {
+                            ...prevUIState,
+                            currOpenTaskpane: {
+                                ...currOpenTaskpane,
+                                openGraph: {
+                                    ...openGraph,
+                                    selectedTab: GraphSidebarTab.Setup
+                                }
+                            }
+                        }
+                    });
+                }
+            },
+            isDisabled: () => {return defaultActionDisabledMessage},
+            searchTerms: ['select data', 'select'],
+            tooltip: "Select the data to be used in the graph."
+        },
+        [ActionEnum.ExportGraphDropdown]: {
+            type: 'build-time',
+            staticType: ActionEnum.ExportGraphDropdown,
+            iconToolbar: ExportIcon,
+            titleToolbar: 'Export',
+            longTitle: 'Open Export Dropdown',
+            actionFunction: () => {
+                setEditorState(undefined);
+                closeOpenEditingPopups();
+
+                setUIState(prevUIState => {
+                    return {
+                        ...prevUIState,
```

Comment:
Do we need to set the currently selected toolbar tab in all of these? I think we do

Reply:
I think the assumption is that if you clicked on the button that triggers this, you're necessarily already in that tab.